Sources/XMTP/ConversationV2.swift

```diff
@@ -30,6 +31,7 @@ public struct ConversationV2 {
 	public var context: InvitationV1.Context
 	public var peerAddress: String
 	public var client: Client
+	public var createdAt: UInt64?
```

Comment:
If this doesn't get accessed externally lets rename this one instead. and make it private. This is just the nanoseconds for when it was created so that may be more descriptive for others using it.
```suggestion
	var createdAtNs: UInt64?
```